integration/hive/src/main/java/org/apache/carbondata/hive/util/HiveCarbonUtil.java

```diff
@@ -155,7 +155,7 @@ public static CarbonLoadModel getCarbonLoadModel(String tableName, String databa
     return loadModel;
   }
 
-  private static TableInfo getTableInfo(String tableName, String databaseName, String location,
+  public static TableInfo getTableInfo(String tableName, String databaseName, String location,
```

Comment:
revert this change after moving getCarbonTable to HiveCarbonUtil

Reply:
done